src/hexo/themes/sonarwhal/layout/about.hbs

```diff
@@ -18,15 +18,17 @@
 
     {{>header navs=site.data.menu}}
     <main class="container">
-        <section class="section">
-            <div class="page-content">
-                {{#compare page.title '===' 'contributors'}}
-                {{>contributors devs=site.data.contributors}}
-                {{else}}
-                {{{page.content}}}
-                {{/compare}}
-            </div>
-        </section>
+        {{#compare page.category '===' 'about-index'}}
+            {{>about-index}}
+        {{else}}
+        {{#compare page.title '===' 'contributors'}}
+        {{>contributors devs=site.data.contributors}}
+        {{else}}
+        <div class="section">
+            {{{page.content}}}
+            {{/compare}}
+            {{/compare}}
+        </div>
```

Comment:
I think the two `{{/compare}}`s should be outside the `</div>`. Can you indent the `compare`s?

Reply:
Will do!